cmd/init.go

```diff
@@ -260,7 +260,7 @@ func askForOptions(options []string, label string) (string, error) {
 		Items: options,
 		Size:  len(options),
 		Templates: &promptui.SelectTemplates{
-			Label:    fmt.Sprintf("%s {{ . }}", log.BlueString("?")),
+			Label:    fmt.Sprintf("{{ . }}"),
```

Comment:
what is this change for?

Reply:
It removes the question mark at the beginning of the label, which looks odd:
<img width="942" alt="Screenshot 2020-06-17 at 11 55 24" src="https://user-images.githubusercontent.com/7474696/84884026-8701a580-b091-11ea-8c39-1b917eafb0ca.png">